docs/DevelopmentPlan/DevelopmentPlan.tex

```diff
@@ -187,19 +187,48 @@ \section{Coding Standard}
 
 \section*{Appendix --- Reflection}
 
-\wss{Not required for CAS 741}
+% \wss{Not required for CAS 741}
 
-\input{../Reflection.tex}
+% \input{../Reflection.tex}
 
 \begin{enumerate}
     \item Why is it important to create a development plan prior to starting the
-    project?
+    project?\\\\
+    Starting with a development plan was useful for our team for two reasons. First, it helped us develop a clearer picture of what our project will look
+    like. In order to make development plan decisions, we had to discuss the actual project itself. This gave us a chance to clear up misconceptions
+    we were having, and dicuss what our overall mission is. Almost more importantly though, working on this document (and problem statement/goals)
+    was a sort of dry run for future deliverables. As we worked on this doc, we were able to put our plans of how we thought we would work together to the test. We found 
+    that our git workflow needed a couple refinements and clarifications. For example, we needed to create more descriptive pull request titles and descriptions
```

Comment:
The writing seems lax in some places lol, is this the essence of the reflection or should it be as professional as the rest of the doc? Depending on this, things like "doc" should be "document". Also "a couple OF refinements"

Reply:
They are looking for sincerity so I went for the more casual/ authentic tone. I will fix the other things